src/libraries/System.Drawing.Common/src/System/Drawing/GdiplusNative.cs

```diff
@@ -1374,8 +1374,11 @@ internal struct StartupInput
 
             public static StartupInput GetDefault()
             {
+                OperatingSystem os = Environment.OSVersion;
                 StartupInput result = default;
-                result.GdiplusVersion = 1;
+
+                // In Windows 7 GDI+1.1 story is different as there are different binaries per GDI+ version.
+                result.GdiplusVersion = os.Platform != PlatformID.Unix && os.Version.Major == 6 && os.Version.Minor == 1 ? 1 : 2;
```

Comment:
Why `os.Platform != PlatformID.Unix` and not `== PlatformID.Win32NT` or `RuntimeInformation.IsOSPlatform`?

Reply:
Also, where is the documentation for GDI+ 1.1, the docs all indicate `GdiplusVersion` must be `1` and I don't see anything about `2`

Reply:
> Why os.Platform != PlatformID.Unix and not == PlatformID.Win32NT or RuntimeInformation.IsOSPlatform?

I can change it to use `PlatformID.Win32NT`. I used `os.Platform` because instead of `RuntimeInformation` because I was already querying `OSEnvironment` to get the os major and minor version.

> Also, where is the documentation for GDI+ 1.1, the docs all indicate GdiplusVersion must be 1 and I don't see anything about 2

I couldn't find it either, but this is what the GDI team recommended and by looking at the code those are the accepted values, 1 is for 1.0 and 2 for 1.1. I can forward you the offline thread with the GDI team.

Reply:
Updated.